global_costmap:
  robot_base_frame: base_link #nazwa podstawy robota
  update_frequency: 10.0 #częstotliwość odświeżania mapy globalnej
  static_map: true #wykorzystanie mapy statycznej, czyli tej z lab3
  plugins: #przestrzenie nazw wtaz z ich typami
    - {name: static_map,       type: "costmap_2d::StaticLayer"}
    - {name: obstacles,        type: "costmap_2d::VoxelLayer"}
    - {name: inflation,        type: "costmap_2d::InflationLayer"}
  static_map:
    map_topic: map #publikowanie mapy na topic
  obstacles:
    observation_sources: base_scan #przeszkody publikowane na topic /base_scan
    base_scan: {data_type: LaserScan, sensor_frame: base_laser_link, clearing: true, marking: true, topic: /scan_raw}
  inflation:
    inflation_radius: 0.3 #wartośc promienia od przeszkód
